apps/cli/src/commands/send/erc20.ts

```diff
@@ -83,8 +83,7 @@ export default class SendERC20 extends SendBaseCommand<typeof SendERC20> {
                 validate: ercValidator,
             })) as Address);
 
-        const amount =
-            this.flags.amount || (await input({ message: "Amount" }));
+        const amount = await input({ message: "Amount", default: this.flags.amount });
```

Comment:
Instead of removing the option ofreading the flag, the right way to fix this is remove the default value of the flag up there in line 29

Reply:
Fixed.